tools/scripts/features/frcnn/processing_image.py

```diff
@@ -69,9 +69,7 @@ def __call__(self, images, single_image=False):
                 elif not isinstance(images[i], torch.Tensor):
                     images.insert(
                         i,
-                        torch.as_tensor(
-                            img_tensorize(images.pop(i), input_format=self.input_format)
```

Comment:
Is the `input_format` arg not required?

Reply:
No its not I simplified the function pretty significantly from what huggingface had, they used a cv2 and some other weird functions that I just omitted, I guess I just forgot to delete that param in this use of the function when I first committed.